app/cafe.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: Dict[str, str]) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError()

```

Comment:
Code Style: It would be better to use a descriptive error message when raising a `NotVaccinatedError`. It helps with debugging and makes the code more understandable. Instead of `raise NotVaccinatedError()`, you could write something like `raise NotVaccinatedError('The visitor has not been vaccinated.')`.